fsw/src_mysql/sadb_routine_mariadb.template.c

```diff
@@ -231,10 +243,45 @@ static int32 parse_sa_from_mysql_query(char* query, SecurityAssociation_t** secu
 
     return status;
 }
+static int32 convert_hexstring_to_byte_array(char* source_str, uint8* dest_buffer)
```

Comment:
I like this function - should we consider moving some of these helpers like this to a utils file / directory for access in other files e.g. crypto_util/src?

Reply:
I don't think the MariaDB/mysql source code should depend on the crypto_util source tree at this time... So far, everything under crypto_util are test & related helper applications/libraries, not things we expect end-users to use.

Reply:
This is an intermediary PR, there is still mysql integration work to do... since this is the only comment and the function can be refactored when needed, I'm going to proceed with the merge.